### PR TITLE
#978 - remove ready() function and related code

### PR DESCRIPTION
**Pull request by treshugart**

### CODE REVIEW

File: test/perf/react.js [hunk 1]
```diff
@@ -1,129 +0,0 @@
-/* eslint-env jasmine, mocha */
```
Comment (treshugart) [inline test/perf/react.js]: This was waaay out of date and used `ready()` so just removed.

File: test/unit/deprecated/lifecycle.js [hunk 2]
```diff
@@ -1,113 +0,0 @@
-/* eslint-env jasmine, mocha */
```
Comment (treshugart) [inline test/unit/deprecated/lifecycle.js]: This used `ready()` and was testing stuff that the custom element polyfill should be tested to perform.

**Comment by Hotell** [comment]
But the main removal is already merged via https://github.com/skatejs/skatejs/pull/1063 right?

**Comment by treshugart** [comment]
@Hotell that just removed one of the tests.

**Pull request merged**